src/content/en/2021/privacy.md

```diff
@@ -5,572 +5,544 @@ description: Privacy chapter of the 2021 Web Almanac covering adoption and impac
 authors: [ydimova, victorlep]
 reviewers: [maudnals]
 analysts: [victorlep, max-ostapenko]
-editors: []
+editors: [tunetheweb]
 translators: []
 results: https://docs.google.com/spreadsheets/d/148SxZICZ24O44roIuEkRgbpIobWXpqLxegCDhIiX8XA/
-ydimova_bio: Yana Dimova is a PhD student at imec-DistriNet, working on web privacy. Her general interests and work focus on online tracking, privacy vulnerabilites and privacy legislation and policies.
-victorlep_bio: Victor Le Pochat is a PhD researcher at the <a hreflang="en" href="https://distrinet.cs.kuleuven.be/">imec-DistriNet</a> research group of KU Leuven in Belgium. His interests lie in the exploration of web ecosystems, and in web security/privacy research metholodogy, both analyzing and improving current methods.
+ydimova_bio: Yana Dimova is a PhD student at imec-DistriNet, working on web privacy. Her general interests and work focus on online tracking, privacy vulnerabilities and privacy legislation and policies.
+victorlep_bio: Victor Le Pochat is a PhD researcher at the <a hreflang="en" href="https://distrinet.cs.kuleuven.be/">imec-DistriNet</a> research group of KU Leuven in Belgium. His interests lie in the exploration of web ecosystems, and in web security/privacy research methodology, both analyzing and improving current methods.
 featured_quote: "Browsers are paving the way for better user privacy&colon; Firefox and Safari have already deployed Tracking Protection, while Chrome is proposing new privacy-protecting technologies that are discussed in the open and can be tested by website owners."
-featured_stat_1: 82.1%
-featured_stat_label_1: Websites that include at least one tracker
-featured_stat_2: 39.7%
-featured_stat_label_2: Websites that contain a privacy policy link
+featured_stat_1: 82.08%
+featured_stat_label_1: Mobile websites that include at least one tracker
+featured_stat_2: 39.70%
+featured_stat_label_2: Mobile websites that contain a privacy policy link
 featured_stat_3: 4.10%
 featured_stat_label_3: Popular sites opting out of FLoC cohorts
-unedited: true
 ---
 
+## Introduction
 
+[_"On the Internet, nobody knows you're a dog."_](https://en.wikipedia.org/wiki/On_the_Internet,_nobody_knows_you%27re_a_dog) While it might be true that you could try to remain anonymous to use the Internet as such, it can be quite hard to keep your personal data fully private. A [whole industry](https://crackedlabs.org/en/corporate-surveillance/) is dedicated to tracking users online, to build detailed user profiles for purposes such as targeted advertising, fraud detection, price differentiation, or even credit scoring.
 
```

Comment:
Isn't the whole chapter about that? 😁

Here's the original: https://20211109t172922-dot-webalmanac.uk.r.appspot.com/en/2021/privacy?print
(I've used the `?print` query param to show the original images, since I changed some of the graphs)

And the new version: https://20211112t221945-dot-webalmanac.uk.r.appspot.com/en/2021/privacy

I think the "breathing space" makes it easier to read. Especially on mobile. But OK to merge some of these back in if you feel strongly about this. Do you want to merge the ones you feel should be re-merged?

Reply:
Everywhere else, the breathing space is an improvement indeed, thanks! Just in this one spot I find that it breaks the flow :) We could break just before "A whole industry", but then the first paragraph might become too short?

Reply:
SGTM